# Python - intro 101

## Lab 1 - Qué va a crear

En este desafío, creará un contador de calorías que solicita al usuario lo siguiente:

- La fecha actual (en cualquier formato)
- Calorías ingeridas en el desayuno
- Calorías ingeridas en la comida
- Calorías ingeridas en la cena
- Calorías ingeridas en tentempiés

Después, el programa sumará todas las calorías y les dará formato de mensaje.

In [7]:
# Creamos las variables del contador
print("¿Qué día es hoy?")
date = input()

¿Qué día es hoy?


 1


In [9]:
date

'1'

In [14]:
print("Calorias desayuno")
breakfast = int(input())

Calorias desayuno


 500


In [15]:
breakfast

500

In [1]:
# Creamos las variables del contador
print("¿Qué día es hoy?")
date = input()
print("Calorias desayuno")
breakfast = int(input())
print("Calorias comida")
lunch = int(input())
print("Calorias cena")
dinner = int(input())
print("Calorias merienda")
snack = int(input())

¿Qué día es hoy?


KeyboardInterrupt: Interrupted by user

In [17]:
totalCal = breakfast + lunch + dinner + snack

In [26]:
totalCal

969

In [27]:
str(totalCal)

'969'

In [31]:
print("El total de las calorías del día " + date + " son: " + str(totalCal) + " calorias.")

El total de las calorías del día 23 marzo 2020 son: 969 calorias.


`Observación`

Hemos observados que las calorías del días 23 de marzo 2020 eran de 969...

In [ ]:
print("Calorias totales del día: " + date + " son: " + str(totalCal))


***
